.travis.yml

```diff
@@ -11,6 +11,7 @@ install:
     - pip install -r dev-requirements.txt
     - pip install -r requirements.txt
     - pip install pandas==0.19 ; python_version==3.5
+    - pip install cython ; python_version==3.6
```

Comment:
try moving `pip install cython ; python_version==3.6` above `pip install pandas==0.19 ; python_version==3.5`

Reply:
I kind of feel like Travis is gaslighting me, because it's now showing that the commit you looked at passed. It was definitely failing before! ¯\\_(ツ)_/¯

Reply:
It definitely is gaslighting you...in the build that passed, it seems to be the latest commit (see image and note that cython is indeed installed before pandas).

![image](https://user-images.githubusercontent.com/25256436/52587921-d6322900-2e00-11e9-8185-e987d541d5cc.png)

I'm guessing the title of the build is off due to whatever issue caused Travis not to build in the first place?